Automattic/vip-go-mu-plugins
# Update Jetpack 11.3 beta3 to 11.3.1 and set default Jetpack version as 11.3

## Description
This PR updates Jetpack 11.3 beta-3 to the official 11.3 and sets the default Jetpack version as 11.3

## Changelog Description

### Plugin Updated: Jetpack 11.3.1

Jetpack 11.3.1 is now the default version if not pinned.
 
## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Pull in PR
2) Go to Jetpack and look at latest version to verify it is `11.3`
